tests/Microcks.Testcontainers.Tests/Async/Kafka/KafkaContainerHelper.cs

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright The Microcks Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+using System.Collections.Generic;
+using System.Text;
+using DotNet.Testcontainers.Configurations;
+using DotNet.Testcontainers.Networks;
+using Testcontainers.Kafka;
+
+namespace Microcks.Testcontainers.Tests.Async.Kafka;
+
+public static class KafkaContainerHelper
```

Comment:
I assume that stuffs from this package comes because the official `KafkaContainer` in Testcontainers cannot expose Kafka on more than one network port? Shouldn't we put this class in `/src` instead of `/tests` to make is distributable while the enhancement is not integrated into Testcontainers? WDYT?

Reply:
Can be changed later on. I'll mark this one as solved to I can merge it.